### PR TITLE
Z7kyrtaq

### PR DESCRIPTION
**Pull request by kyrtaq**
zadanie 7 gotowe

### CODE REVIEW

**Comment by kyrtaq** [comment]
cos mi sie za duzo tych plikow dodalo

File: z7/main.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+int main() {
+    std::vector<std::string> str{"Ala ", "ma ", "ma ", "kota, ", "a ", "kot ", "ma ", "Ale ", "Ale.. ", "Ala ", "Ala "};
+    std::cout << "BEFORE REMOVING DUPLICATES:\n";
+    print(str);
+    std::cout << "AFTER REMOVING DUPLCAITES:\n";
+    removeDuplicates(str);
+    print(str);
+}
```
Comment (NG90) [inline z7/main.cpp:20]: add new line

File: z7/remove_repeating.cpp [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#include "remove_repeating.hpp"
+
+#include <algorithm>
+
+void removeDuplicates(std::vector<std::string>& str) {
+    std::transform(begin(str), end(str), begin(str), [](std::string& s) {
+        s.erase(std::remove_if(s.begin(), s.end(),
+                               [](unsigned char ch) {
+                                   return ch == '.';
```
Comment (NG90) [inline z7/remove_repeating.cpp:9]: why do you remove just '.', not every punct like ",/?"?

File: z7/remove_repeating.hpp [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+void removeDuplicates(std::vector<std::string>&);
```
Comment (NG90) [inline z7/remove_repeating.hpp:6]: this function should take std::string as an argument, not std::vector<std::string>

File: z7/remove_repeating.cpp [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#include "remove_repeating.hpp"
+
+#include <algorithm>
+
+void removeDuplicates(std::vector<std::string>& str) {
```
Comment (NG90) [inline z7/remove_repeating.cpp:5]: this function should take std::string as an argument, not std::vector<std::string>.
In the first step you should probably convert string to the vector of words

**Pull request closed without merging**